### PR TITLE
test: Example for relational API

### PR DESCRIPTION
**Pull request by krlmlr**

### CODE REVIEW

**Comment by krlmlr** [comment]
This will auto-merge, I hope. Tagged to review for reference.

I also pushed to main before that.

File: r/tests/testthat/test-base-eq.R [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# To be generated from test.R
+
+test_that("int + int", {
+  con <- con(); on.exit(dbDisconnect(con, shutdown=TRUE))
```
Comment (romainfrancois) [inline r/tests/testthat/test-base-eq.R:4]: I have a `local_con()` test fixture in #14 that we can use instead of manual `on.exit()`

**Review by romainfrancois — APPROVED** [review]
This seems less hacky than what I have in #14, I'll adapt my PR once this is merged. 

Thanks for the pointers to `rep_project()` etc ...

**Comment by romainfrancois** [comment]
> This will auto-merge, I hope.

This does not seem to do the trick. Can we merge manually then ?

**Pull request merged**